modules/dxp/apps/osb/osb-faro/osb-faro-mock-engine-client/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
 	compileOnly group: "com.liferay.portal", name: "release.dxp.api"
-	compileOnly group: "org.springframework", name: "spring-core", version: "5.3.14"
+	compileOnly group: "org.springframework", name: "spring-core", version: "5.3.27"
```

Comment:
Hey @regiCesar21, shouldn't it be the same version as above? Also, shouldn't we update https://github.com/liferay/com-liferay-osb-faro-private/pull/4680/commits/e9810923fdb3db7c0ffa4e6764f347a158f5e538 as well?

Reply:
Yes, should be the same, I changed now here, thanks @marcosapmf